test/sideinput-source-e2e/testdata/sideinput_source.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: sideinput-source
```

```suggestion
  name: sideinput-source-test
```

Just to be consistent with `sideinput-sink-test`.